src/components/institution_table/components/InstitutionDomainEditor.tsx

```diff
@@ -20,6 +21,49 @@ export const InstitutionDomainEditor = ({ domains, isEditing, onDomainsChange, i
         });
     });
 
+    const validateDomain = (domain: string): string | null => {
+        try {
+            // Check if it's a valid FQDN (Fully Qualified Domain Name)
+            if (!validator.isFQDN(domain, {
```

Comment:
Since we're only using `validator` here, instead of pulling in another dependency would it be better to copy the method over? https://github.com/validatorjs/validator.js/blob/master/src/lib/isFQDN.js#L13

Reply:
Greg [suggests](https://github.com/DataBiosphere/duos-ui/pull/2985#discussion_r2228416423) using `validator` for more email validation as well.  Think it's worthwhile soon?  If so, it might be most efficient to keep this dependency.